cloudformation/media-atom-maker-dev.yml

```diff
@@ -240,17 +240,26 @@ Resources:
                     - "arn:aws:dynamodb:${Region}:${Account}:table/${Table}"
                     - { Region: !Ref "AWS::Region", Account: !Ref "AWS::AccountId", Table: !Ref "UploadTrackingDynamoTable" }
               - Effect: "Allow"
-                Action: ["s3:GetObject"]
+                Action: ["s3:*"]
```

Comment:
To allow the deletion of entries once uploads are complete. I'll make the permissions more specific to be clearer